src/Miner/Miner.cpp

```diff
@@ -98,6 +100,7 @@ void Miner::workerFunc(const BlockTemplate& blockTemplate, uint64_t difficulty,
       CachedBlock cachedBlock(block);
       Crypto::Hash hash = cachedBlock.getBlockLongHash();
       if (check_hash(hash, difficulty)) {
+	std::cout << InformationMsg("Found block for difficulty ")  << difficulty  << std::endl;
```

Comment:
We should not write directly to std::cout, whe have the `Logging::ILogger` interface for that. May consider extending the interface if it does not fit your need.

Reply:
Okay, good point, if we are wanting to use the ILogger I'll use that class for the future.